src/resolvers/hosts/hosts.integration.ts

```diff
@@ -530,7 +530,7 @@ describe('hosts query', function () {
                 // eslint-disable-next-line @typescript-eslint/no-unused-vars
                 test.each(test_data)('$field_name $field_query', async ({field_name, field_query}) => {
                     const { data } = await runQuery(BASIC_QUERY, {filter: field_query});
-                    data.hosts.data.should.have.length(1);
+                    data.hosts.data.should.have.length <= 2;
```

Comment:
Why this change? I just want to make sure this was changed for a good reason and not just to force the test to pass.
afaict this test is meant to validate that exactly one host is returned for each of the filters; if one of them is returning more than one host, that implies to me that filter isn't working correctly

Reply:
As far as I understood that part, it loops over the `field_query` data from the `spf_test_data.json` file and most of the field have only 1 occurence, except the two new `bootc_status` and `intersystems` fields added by the automation.

For example:
![image](https://github.com/RedHatInsights/xjoin-search/assets/242115/ee6b43af-63eb-4343-867c-6ed22f5955c7)

Reply:
And to answer the primary question, that is the only manual fix I did here in this PR, the rest has been done by running the automation script locally.

Reply:
If it helps, the IDs of the hosts being returned are:

6e7b6317-0a2d-4552-a2f2-b7da0aece49d
22cd8e39-13bb-4d02-8316-84b850dc5136
f5ac67e1-ad65-4b62-bc27-845cc6d4bcee
f5ac67e1-ad65-4b62-bc27-845cc6d4bc01
f5ac67e1-ad65-4b62-bc27-845cc6d4bc02
f5ac67e1-ad65-4b62-bc27-845cc6d4bc03
f5ac67e1-ad65-4b62-bc27-845cc6d4bc04